data-structures/README.md

```diff
@@ -0,0 +1,5 @@
+# Data structures
```

Comment:
Preferred folder name is **Linked Lists** , link we have different folder for strings, the entire repository is about data structures and algorithms.

Reply:
Thanks for the reply i'am new so do i have to make changes and create new pull request?

Reply:
You don't need to create new pull requests. Just add more commits to the source branch in your fork (in this case, main), and the commits will be automatically added to the respective pull request.